### PR TITLE
[WIP] Fixed travis memory limit for composer

### PR DESCRIPTION
**Pull request by iltar**
Do not merge yet.

### CODE REVIEW

**Comment by iltar** [comment]
Okay, no idea how to use it :sweat_smile: I'll fix it next week.

File: .travis.yml [hunk 1]
```diff
@@ -36,6 +37,6 @@ before_install:
     - if [[ "$SYMFONY_VERSION" != "" ]]; then composer require --no-update symfony/symfony:${SYMFONY_VERSION}; fi
     - if [[ "$WITHOUT_EXTRA_BUNDLE" != "" ]]; then composer remove "sensio/framework-extra-bundle" --dev; fi;
 
-install: composer update $COMPOSER_FLAGS --prefer-dist
+install: composer $COMPOSER_MEMORY_LIMIT update $COMPOSER_FLAGS --prefer-dist
```
Comment (xabbuh) [inline .travis.yml]: just revert the change here

Comment (iltar) [reply .travis.yml]: You're probably right. I tried adding it here because of this build: https://travis-ci.org/hostnet/form-handler-bundle/builds/325502642

However, this build has overwritten the env vars. I think I should just properly set the env var at this particular point, to ensure it's always set.

**Comment by iltar** [comment]
That should do the trick, it will now always run with 2G.
